src/runtime_src/tools/scripts/pkgapu.sh

```diff
@@ -177,7 +178,6 @@ if [[ ! -d $IMAGES_DIR ]]; then
 	error "Please specify the valid path of APU images by -images"
 fi
 IMAGES_DIR=`realpath $IMAGES_DIR`
-source /proj/xbuilds/2022.2_released/installs/lin64/Vitis/2022.2/settings64.sh
```

Comment:
Similarly, we should keep this here since pkgapu.sh could potentially be called independently.
And source it from vitis.build env file.

Reply:
Makes sense! Thanks for the suggestion :)